ci/asan_tests/bazelrc

```diff
@@ -0,0 +1,26 @@
+# Cache
```

Comment:
If `root/.bazelrc` changes ASAN options, this must be also changed right? Would you mind adding comments at both files to explain that?